toonz/sources/toonz/xshrowviewer.cpp

```diff
@@ -830,6 +849,15 @@ void RowArea::paintEvent(QPaintEvent *event) {
     // current frame
     drawCurrentRowGadget(p, r0, r1);
 
+#ifdef WITH_STOPMOTION
+  StopMotion *stopMotion = StopMotion::instance();
+  if (stopMotion->getPlaceOnXSheet() &&
+      (stopMotion->m_usingWebcam || stopMotion->m_active)) {
+    drawStopMotionCameraIndicator(p);
```

Comment:
I'm not sure with the DSLR case, but when using webcams IMO the indicator can be hidden if the live view is deactivated.